rpcclient/evo_test.go

```diff
@@ -106,12 +127,28 @@ func TestQuorumDKGStatus(t *testing.T) {
 	}
 	defer client.Shutdown()
 
-	for _, dl := range []btcjson.DetailLevel{
-		btcjson.DetailLevelCounts,
-		btcjson.DetailLevelIndexes,
-		btcjson.DetailLevelMembersProTxHashes,
-	} {
+	testCases := []struct {
+		detailLevel btcjson.DetailLevel
+		response    interface{}
+	}{
+		{
+			detailLevel: btcjson.DetailLevelCounts,
+			response:    btcjson.QuorumDKGStatusCountsResult{},
+		},
+		{
+			detailLevel: btcjson.DetailLevelIndexes,
+			response:    btcjson.QuorumDKGStatusIndexesResult{},
+		},
+		{
+			detailLevel: btcjson.DetailLevelMembersProTxHashes,
+			response:    btcjson.QuorumDKGStatusMembersProTxHashesResult{},
+		},
+	}
+	for _, tc := range testCases {
+		dl := tc.detailLevel
 		t.Run(fmt.Sprint(dl), func(t *testing.T) {
+			client.httpClient.Transport = mockRoundTripperFunc(&tc.response)
+
```

Comment:
`compareWithCliCommand` does not do anything, we need to verify `result` somehow.